di_website/common/templatetags/string_utils.py

```diff
@@ -50,3 +51,15 @@ def richtext_strip_wrapper(value):
         html = expand_db_html(value)
 
     return mark_safe(html)
+@register.filter(expects_localtime=True, is_safe=False)
+def event_date(start, end=None):
```

Comment:
Is this function going to be used on non-event related stuff? If so, then rename to something more generic and descriptive of what it actually does. Otherwise move to a local template tag in the event module.